script/generate-config-gypi.py

```diff
@@ -60,6 +61,9 @@ def main(target_file, target_cpu):
   v['node_module_version'] = int(args['node_module_version'])
   # Used by certain versions of node-gyp.
   v['build_v8_with_gn'] = 'false'
+  # TODO(zcbenz): Support the flags in Node's configure script.
+  is_arm = get_target_arch().startswith('arm')
```

Comment:
Should be disabled for all 32-bit builds, `arm` and  `ia32`

Reply:
Would it be better to patch the 32-bit block https://github.com/zcbenz/node/blob/2a1bd660bd15f19da8cea50ef7449723ac55bdc5/common.gypi#L105-L109 as it was done before in build_ensure_v8_pointer_compression_sandbox_is_enabled_on_64bit.patch and upstream it ?

Reply:
@deepak1556 agreed - done! pushed it up.